kobo/apps/organizations/utils.py

```diff
@@ -30,10 +30,17 @@ def get_monthly_billing_dates(organization: Union[Organization, None]):
         ):
             return first_of_this_month, first_of_next_month
         
-        period_end = canceled_subscription_anchor.replace(tzinfo=pytz.UTC)
+        canceled_subscription_anchor = canceled_subscription_anchor.replace(
+            tzinfo=pytz.UTC
```

Comment:
Let's use ZoneInfo and remove dependency on `pytz`

Reply:
Done. I also removed it from kobo/apps/subsequences/actions/base.py for good measure so it isn't in our code base anymore aside from being a dependency for pandas and flower.